src/formatting/syntux/session/ignore_path.rs

```diff
@@ -42,22 +42,48 @@ mod test {
 
     use super::IgnorePathSet;
     use crate::config::{Config, FileName};
+    use crate::is_nightly_channel;
 
     #[test]
     fn test_ignore_path_set() {
-        match option_env!("CFG_RELEASE_CHANNEL") {
-            // this test requires nightly
-            None | Some("nightly") => {
-                let config =
-                    Config::from_toml(r#"ignore = ["foo.rs", "bar_dir/*"]"#, Path::new(""))
-                        .unwrap();
-                let ignore_path_set = IgnorePathSet::from_ignore_list(&config.ignore()).unwrap();
-
-                assert!(ignore_path_set.is_match(&FileName::Real(PathBuf::from("src/foo.rs"))));
-                assert!(ignore_path_set.is_match(&FileName::Real(PathBuf::from("bar_dir/baz.rs"))));
-                assert!(!ignore_path_set.is_match(&FileName::Real(PathBuf::from("src/bar.rs"))));
-            }
-            _ => {}
-        };
+        if !is_nightly_channel!() {
+            // This test requires non-nightly
```

Comment:
Nit, but want to avoid a confusing comment :wink: 
```suggestion
            // This test requires nightly
```